src/components/Header/UserToggle.tsx

```diff
@@ -122,8 +122,10 @@ const UserToggle = ({ isSmall = false, extraItems = [] }: UserToggleProps) => {
     };
   });
 
-  const onSelect = () => {
-    setIsOpen(!isOpen);
+  const onSelect = (event: any) => {
+    if (event.target.tagName === 'A') {
```

Comment:
We're also using `button` on the logout. Might be better to have `['BUTTON', 'A'].includes(event.target.tagName)` to cover both usecases.